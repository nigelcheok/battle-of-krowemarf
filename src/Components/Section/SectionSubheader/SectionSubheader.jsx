import React from 'react';

export function SectionSubheader(props) {
  return (
    <div style={SectionSubheaderStyle()}>
      { props.text }
    </div>
  );
}

function SectionSubheaderStyle() {
  return {
    marginTop: '2.25rem',
    // marginBottom: '.5rem',
    fontSize: '1.5rem',
    color: 'rgb(37, 39, 41)',
    fontWeight: '600',
  }
}
