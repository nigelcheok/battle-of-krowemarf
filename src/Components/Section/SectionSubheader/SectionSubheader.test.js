import React from 'react';
import { shallow } from 'enzyme';
import { SectionSubheader } from './SectionSubheader';

describe('SectionSubheader', () => {
  it('should render correctly with no props', () => {
    const component = shallow(<SectionSubheader/>);
    expect(component).toMatchSnapshot();
  });

  it('should render correctly with props', () => {
    const component = shallow(<SectionSubheader text="test"/>);
    expect(component).toMatchSnapshot();
  });
});