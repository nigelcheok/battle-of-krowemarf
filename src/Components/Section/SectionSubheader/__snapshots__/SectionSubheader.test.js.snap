// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionSubheader should render correctly with no props 1`] = `
<div
  style={
    Object {
      "color": "rgb(37, 39, 41)",
      "fontSize": "1.5rem",
      "marginTop": "2.25rem",
    }
  }
/>
`;

exports[`SectionSubheader should render correctly with props 1`] = `
<div
  style={
    Object {
      "color": "rgb(37, 39, 41)",
      "fontSize": "1.5rem",
      "marginTop": "2.25rem",
    }
  }
>
  test
</div>
`;
